src/Engine/Manager/RenderManager.cpp

```diff
@@ -268,7 +268,38 @@ void RenderManager::Render(World& world, const glm::mat4& viewMatrix, const glm:
     }
     renderSurface->GetShadingFrameBuffer()->Unbind();
 
-    /// @todo Render skinned meshes.
+    // Render skinned meshes.
+    renderSurface->GetShadingFrameBuffer()->BindWrite();
+    { PROFILE("Render skinned meshes");
+    { GPUPROFILE("Render skinned meshes", Video::Query::Type::TIME_ELAPSED);
+    { GPUPROFILE("Render skinned meshes", Video::Query::Type::SAMPLES_PASSED);
+
+        // Cull lights and update light list.
+        LightWorld(world, viewMatrix, projectionMatrix, viewProjectionMatrix);
+
+        // Push matricies and light buffer to the GPU.
+        renderer->PrepareSkinnedMeshRendering(viewMatrix, projectionMatrix);
+
+        // Render meshes
+        /// @todo Sort meshes after animation controller instead of
+        /// meshes would be better.
+        for (Mesh* mesh : meshComponents) {
+            if (mesh->IsKilled() || !mesh->entity->enabled)
+                continue;
+
+            if (mesh->geometry != nullptr && mesh->geometry->GetType() == Video::Geometry::Geometry3D::SKIN) {
+                Entity* entity = mesh->entity;
+                Material* material = entity->GetComponent<Material>();
+                AnimationController* controller = entity->GetComponent<AnimationController>();
+                if (material != nullptr && controller != nullptr && controller->skeleton != nullptr) {
+                    renderer->RenderSkinnedMesh(mesh->geometry, material->albedo->GetTexture(), material->normal->GetTexture(), material->metallic->GetTexture(), material->roughness->GetTexture(), entity->GetModelMatrix(), controller->bones, false);
+                }
+            }
+        }
+    }
+    }
+    }
+    renderSurface->GetShadingFrameBuffer()->Unbind();
     
     // Anti-aliasing.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetAnimations' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)